src/logic/currentSession/store/reducer/currentSession.ts

```diff
@@ -7,24 +6,23 @@ import { saveCurrentSessionToStorage } from 'src/logic/currentSession/utils'
 
 export const CURRENT_SESSION_REDUCER_ID = 'currentSession'
 
-export type SerializedSessionState = {
-  viewedSafes: string[]
-}
+export type SerializedSessionState = string[]
```

Comment:
What are we serializing there?

Reply:
Same as before

Reply:
It's not serialized, previously we would prefix the type with serialized if the state would be an immutable.js js data structure and we would convert it to json